src/_pytest/fixtures.py

```diff
@@ -31,6 +33,9 @@
 from _pytest.outcomes import fail
 from _pytest.outcomes import TEST_OUTCOME
 
+if False:  # TYPE_CHECKING
+    from typing import Type
```

Comment:
Why is this only imported conditionally, but not the others?

Reply:
it is not available in python 3.5.1 / 3.5.2